docs/examples/normal_distribution.rst

```diff
@@ -0,0 +1,59 @@
+Normal Distribution
+===================
+
+The normal distribution, also known as the Gaussian distribution, is a continuous probability distribution characterized by its bell-shaped curve, symmetric around the mean. It is defined by two parameters: the mean (μ) and the standard deviation (σ). The mean determines the center of the distribution, while the standard deviation controls the spread or width of the distribution.
+
+The normal distribution is often the result of summing many small, independent effects, a concept encapsulated by the Central Limit Theorem. Many biological processes exhibit this property, such as human height, blood pressure, and measurement errors. In such cases, numerous genetic, environmental, and random factors contribute to the observed outcome, each adding a small effect. For example, human height is influenced by multiple genes and environmental factors, which together result in a normal distribution of heights across a population.
+
+Normal priors are commonly chosen in Bayesian analysis because they represent a state of limited prior knowledge or weak information. When we assume a parameter has finite variance but lack strong prior knowledge, the normal distribution becomes a suitable choice. This is due to the normal distribution's maximum entropy property among all distributions with a specified mean and variance, ensuring it introduces the least amount of additional assumptions. 
+
+Probability Density Function (PDF):
+____________________________________________________
+
+.. image:: normal_pdf.png
```

Comment:
We should include the code to generate the images (instead of the images), or we should include somewhere the instructions to generate new images, like using "arviz-docs" style or whatever..